src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/PseudonymEntity.kt

```diff
@@ -60,6 +61,22 @@ class PseudonymEntity(
   var version: Int = 0,
 ) {
   companion object {
+    fun from(person: Person): List<PseudonymEntity> {
+      val middleNamesAsString = person.middleNames?.joinToString(" ") { it }
+      val primary = PseudonymEntity(
+        firstName = person.firstName,
+        middleNames = middleNamesAsString,
+        lastName = person.lastName,
+        nameType = NameType.PRIMARY,
+        title = person.title,
```

Comment:
title is not mentioned in the ticket, should it also be moved? It feels more likely a candidate than date of birth but we should check with BAs